docs/UserGuide.md

```diff
@@ -187,19 +189,26 @@ Format: `addEvent INDEX ed/EVENT_DESCRIPTION da/DATE [ti/TIME] [du/DURATION] [t/
 
 
 Examples:
-* `addSchedule 3 da/20-12-2022 ti/10:00 du/2H30M`
-* `addSchedule 2 da/22-11-2023 ti/12:00`
+* `addSchedule 3 da/2022-12-20 ti/10:00 du/2H30M`
```

Comment:
Should the command should be addEvent here?

Reply:
you are right